app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/ui/browser/BrowserFragment.java

```diff
@@ -154,4 +155,31 @@ public void onFocusChanged(boolean focus) {
             }
         });
     }
+
+    private void initNavigationBar(NavigationBar navigationBar) {
+        navigationBar.inflateMenu(R.menu.navigation, getActivity().getMenuInflater());
+        navigationBar.setOnMenuItemClickListener(new ActionMenuView.OnMenuItemClickListener() {
+            @Override
+            public boolean onMenuItemClick(MenuItem item) {
+                switch (item.getItemId()) {
+                    case R.id.action_back:
+                        browserPresenter.navigateHistoryBackward();
+                        return true;
+                    case R.id.action_forward:
+                        browserPresenter.navigateHistoryForward();
+                        return true;
+                    case R.id.action_share:
```

Comment:
Let's remove this and implement features in their entirety. We can add the share button when the app has share functionality.